pytest_rts/pytest/normal_phase_plugin.py

```diff
@@ -27,6 +21,8 @@ def pytest_collection_modifyitems(self, session, config, items):
         for item in items:
             if item.nodeid in self.test_set:
                 selected.append(item)
+                if item.nodeid not in self.test_func_times:
+                    self.test_func_times[item.nodeid] = sys.maxsize
```

Comment:
This code needs refactoring; too many nested levels. Instead of loop use comprehension and/or higher level data structure operations (like set intersection). The rule of thumb: if you have a loop in Python code, think twice if it should be there :) Also you almost never need `.append()` method; most of the time one is better using either comprehension directly, or comprehension + `.extend()`.